### DISCUSSION TITLE
dead link in tutorial, 04_Estimating_model_parameters.html

### DISCUSSION THREAD

**Discussion by mb6852** [moj-analytical-services/splink#2424] [General]
this [link]([here](https://moj-analytical-services.github.io/splink/linkerest.html)) on page [here](https://moj-analytical-services.github.io/splink/demos/tutorials/04_Estimating_model_parameters.html), is dead.

**Comment by RobinL** [comment]
Thanks for the report, fixed [here](https://github.com/moj-analytical-services/splink/pull/2430)
